backend/scraper/mapping_police_violence.py

```diff
@@ -162,12 +167,26 @@ def make_all_tables():
         sub_table = full_df
         table_list.append(sub_table)
     table_dict = {table_names[i]: table_list[i] for i in range(len(table_list))}
+    df_victim = table_dict["victim"]
+    victim_indices = df_victim['victim_name_full'] != 'Name withheld by police'
+    victim_indices_2 = df_victim['victim_name_full'] != ''
+    victim_indices_3 = df_victim['victim_name_full'].notnull()
+    filtered_indices = (victim_indices & victim_indices_2 & victim_indices_3)
+    df_victim_filtered = pd.concat([
+        df_victim[df_victim['victim_name_full'] == 'Name withheld by police'],
+        df_victim[df_victim['victim_name_full'] == ''],
+        df_victim[df_victim['victim_name_full'].isnull()],
+        df_victim[filtered_indices].drop_duplicates(['victim_name_full'],
+                                                    keep='first')
+    ])
+    final_indices = df_victim_filtered.index
+
     writer = pd.ExcelWriter('full_database.xlsx', engine='xlsxwriter')
 
     # Write each dataframe to a different worksheet.
     for key in table_dict:
         sub_data = table_dict[key]
-        sub_data.to_excel(writer, sheet_name=key)
+        sub_data.iloc[final_indices].to_excel(writer, sheet_name=key)
```

Comment:
I think this should use `.loc` rather than `.iloc`, since the index is a list of row labels, while `iloc` refers to the integer location of the row. I think it might work in this case due to our choice of index but that could change in the future.